x-pack/plugins/security_solution/public/resolver/view/resolver_no_process_events.tsx

```diff
@@ -20,36 +23,55 @@ const StyledEuiFlexGroup = styled(EuiFlexGroup)`
   margin: 60px auto 0;
 `;
 
-export const ResolverNoProcessEvents = () => (
-  <StyledEuiFlexGroup data-test-subj="resolver:no-process-events" direction="column">
-    <EuiTitle>
-      <h4>
-        {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.title', {
-          defaultMessage: 'No Process Events Found',
-        })}
-      </h4>
-    </EuiTitle>
-    <EuiSpacer size="m" />
-    <EuiText size="s">
-      {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.timeRange', {
-        defaultMessage: `
+export const ResolverNoProcessEvents = () => {
+  const originTimestamp = useSelector(selectors.resolverTreeOriginTimestamp);
+  const formattedDate = useFormattedDate(originTimestamp);
+  return (
+    <StyledEuiFlexGroup direction="column">
+      <EuiTitle>
+        <h4>
+          {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.title', {
+            defaultMessage: 'No Process Events Found',
+          })}
+        </h4>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      <EuiText size="s">
+        {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.timeRange', {
+          defaultMessage: `
             The Analyze Event tool creates graphs based on process events.
             If the analyzed event does not have an associated process in the current time range,
-            or stored in Elasticsearch within any time range, a graph will not be created.
-            You can check for associated processes by expanding your time range.
-          `,
-      })}
-    </EuiText>
-    <EuiSpacer size="m" />
-    <EuiText size="s">
-      {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.eventCategory', {
-        defaultMessage: `You may also add the below to your timeline query to check for process events.
-            If none are listed, a graph cannot be created from events found in that query.`,
-      })}
-    </EuiText>
-    <EuiSpacer size="m" />
-    <StyledEuiCodeBlock language="html" paddingSize="s" isCopyable>
-      {'event.category: "process"'}
-    </StyledEuiCodeBlock>
-  </StyledEuiFlexGroup>
-);
+            or stored in Elasticsearch within any time range, a graph will not be created.`,
+        })}
+      </EuiText>
+      <EuiSpacer size="m" />
+      {formattedDate ? (
+        <>
+          <EuiText size="s" data-test-subj="resolver:origin-timestamp">
+            {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.originTimestamp', {
+              defaultMessage: `Analyzer did find a process event associated with this event, make sure your current time range
```

Comment:
Missing a `not` in the message. It should be`Analyzer did not find a process...`

Reply:
It would be a good idea to get the copy reviewed as well.

Reply:
Ah, I see. I am not sure then if the copy could be simplified for a novice like me to get it on a first read. The second paragraph could be something like

```
The analyzer found one process event for the following date. Make sure your time range is inclusive of this in order to ...
```

However, if this is meant for people who already know what this is then your current copy is great. 👍